src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const ArticleCard = ({ article }) => {
+  return (
+    <li className="article">
+      <h2>{article.title}</h2>
+      <img src={article.article_img_url} />
```

Comment:
The img tag should include an alt attribute specifying an alternate text for image for anyone using a screen reader